js/util/urlParser.js

```diff
@@ -118,6 +130,11 @@ var urlParser = {
     } else {
       return encodeURI('file://' + path)
     }
+  },
+  isHTTPSUpgreadable: function (url) {
+    let domain = /^([^\/$]+)/.exec(urlParser.removeProtocol(url))
```

Comment:
I'm not sure why there's a $ inside the regex here: if you had something like `example$.com/abc`, it matches "example". I don't think that could ever realistically happen, but probably still worth fixing.

Reply:
Yes, you are right, I don't know what that `$` is doing there. Will fix it